content/en/apps/guides/debugging/Securely-sharing-your-development-environment.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Securely sharing your development environment"
+linkTitle: "Securely sharing your development environment"
+weight: 
+description: >
+  Use a publicly accessible Linux web server to forward https requests to your development environment
+relatedContent: >
+  
+---
+
+## Overview
+When using a local [development environment](https://github.com/medic/cht-core/blob/master/DEVELOPMENT.md), you may want to share your work with other collaborators. You also may need to access the environment from your [mobile device](https://github.com/medic/medic-android/) which requires an SSL certificate (the "s" in "https"). By using a publicly accessible web server, you can receive the secure https requests and forward them back to your CHT instance which doesn't have https set up:
+
+[<img src="/apps/guides/debugging/images/SSH.tunnel.diagram.svg" width=100% height=100%>](/apps/guides/debugging/images/SSH.tunnel.diagram.svg)
+
+Once you have this web server set up, you many continue to use when ever you want by simply reconnecting to it via the secure tunnel.
+
+## Prereqs
+This guide assume:
+* You have a local dev instances set up of cht-core https://github.com/medic/cht-core/blob/master/DEVELOPMENT.md
+* You have medic-android repo checked out and can compile the android app from scratch https://github.com/medic/medic-android/
+* You  have the generic Medic Mobile app installed on your android device https://play.google.com/store/apps/details?id=org.medicmobile.webapp.mobile&hl=en_US
+* You have an Ubuntu >18.04 server with a public IP and a DNS entry that you can SSH into and have sudo on
```

Comment:
Would '>=' to be more precise here?

Reply:
Hi @andy5995! Thanks so much for your feedback on this PR.

I accepted your others, but this one felt a little awkward.  Checking other instances of versions in this repo, they do "use X.X.X or later".  I did a light refactor of this sentence to do the same in 609baf2

Reply:
You're welcome @mrjones-plip!